synfig-studio/src/gui/resources/css/synfig.css

```diff
@@ -28,6 +28,13 @@
 	padding: 5px;
 }
 
+/*
+ * TreeView header buttons of docks are set with a consistent padding
+ */
+.synfigstudio-efficient-workspace treeview.view header button {
+	padding: 5px;
+}
```

Comment:
Are there any TreeView that sets it via code?

Reply:
Currently the style of header is assigned from `synfig.css` using the styling set under `.synfigstudio-efficient-workspace button` which sets padding to 0px.

This part is specific for buttons in `Gtk::TreeView` as in the case of our dock header labels.

From what I remember I didn't find places in the code where this has been explicitly defined and thus I assume we always relied on default Gtk styles.

Reply:
I did not notice any changes. Can you make a screenshot showing what exactly has changed?

Reply:
@ice0 it seems this issue is highly dependent on the OS you're using.

Here on my openSUSE this is how the MetaData dock looks with current master:
![metadata-dock-before](https://user-images.githubusercontent.com/6705690/120309334-6a2ba580-c2d5-11eb-83f8-f5d8b5514cc6.png)
and this is with the pull request:
![metadata-dock-after](https://user-images.githubusercontent.com/6705690/120309605-b545b880-c2d5-11eb-9851-3d2d8ec783c5.png)

The same is applied to remaining docks listed in the issue report.

Reply:
@FirasH if you use the default padding values instead of setting all sides to 5px you wouldn't need to touch Parameters dock file. That class wasn't styled because it has no toolbar buttons.

You can get the default values of treeview buttons padding using gtk inspector. Here's a screenshot:

![Screenshot_2021-06-04_09-05-44](https://user-images.githubusercontent.com/24214509/120742413-718cc200-c514-11eb-8cc7-ccac9e25d852.png)

Reply:
To learn more about GTK inspector 👉🏽 https://blog.gtk.org/2017/04/05/the-gtk-inspector/